benchmarks/config.toml

```diff
@@ -1,9 +1,7 @@
 # Excludes
 EXCLUDES = [
+    "turing_benchmark.jl",
```

Comment:
Similar to above, maybe remove this since we can now use BenchmarkTools' tag to select benchmarks.

Reply:
I think we'd better keep this:

1. if there are broken benchmarks (by broken I mean it will error even only load it), we can use this to temporarily ignore the file, then wait for someone to fix it
2. some helper files, like "tutring_benchmark.jl" and "BenchmarkHelper.jl", are not necessary to load in the process which benchmarks are running, for performance and for pkg dependency concerns, we'd better ignore them as well.